docs/howtoguides/how_to_know_if_system_affected_by_a_list_of_cves.rst

```diff
@@ -0,0 +1,52 @@
+.. _pro-check-list-cves:
+
+Given a list of CVEs, how to know if you are affected by them?
+**************************************************************
+
+If you have a list of Common Vulnerabilities and Exposures (CVE) or Ubuntu Security Notifications (USN) and want to check if your Ubuntu system is affected by them, you can easily check using the pro fix API. In this guide we'll show you how to leverage the pro fix API:
+
+.. note::
+    The `pro fix` API is provided as a part of the `pro` package, which is a security tool for Ubuntu systems. If you don't have it installed, you can install it by following the instructions in the `pro` package documentation.
+
+Using the `pro.fix.cve.plan` API
+--------------------------------
+
+To check if your system is affected by a list of CVEs, you would need to use the ``u.pro.security.fix.cve.plan.v1`` endpoint as the
+CLI `pro fix --dry-run` command does not support multiple CVEs. However, as can be seen :ref:`in the endpoint documentation <cve-execute-api-v1>`,
+this API endpoint will output a JSON blob containing the current status of each CVE.
```

Comment:
```suggestion
To check if your system is affected by a list of CVEs, you need to use the ``u.pro.security.fix.cve.plan.v1`` API endpoint since the
``pro fix --dry-run`` CLI command is only used to check individual CVEs. This endpoint will output a JSON blob containing the current status of each CVE, as can be seen :ref:`in the endpoint documentation <cve-execute-api-v1>`.
```